components/rebuild/src/lib.rs

```diff
@@ -194,6 +194,8 @@ fn handle_page_editing(site: &mut Site, path: &Path) -> Result<()> {
         Some(prev) => {
             // Front matter didn't change, only content did
             if site.pages[path].meta == prev.meta {
+                // We need to update the page object in the section so it will render correctly
+                site.populate_sections();
                 // Other than the page itself, the summary might be seen
                 // on a paginated list for a blog for example
                 if site.pages[path].summary.is_some() {
```

Comment:
Probably need to change this condition, since the summary may have changed from something -> nothing, or even the full text might be used on a paginated list (my previous blog did this).